.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,9 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Security Vulnerabilities
+    about: See the O3DE security policy for reporting guidance
```

Comment:
Can we also say something like "Please do not submit a GitHub issue to report a suspected security problem."? Maybe having this as a contact link is enough, but I just want us to be super clear that these reporting guidelines are an "instead of" thing vs. an _and_ thing w/r/t cutting an issue.

Reply:
Actually, I retract this ask b/c adding all that to the link description is probably too much.